autopush/settings.py

```diff
@@ -1,7 +1,7 @@
 """Autopush Settings Object and Setup"""
 import socket
 
-from cryptography.fernet import Fernet
+from cryptography.fernet import Fernet, MultiFernet
```

Comment:
Well, it's not as rad as I'd prefer. All it does is create a list of Fernets that encrypt() or decrypt() runs through. I might monkey patch it in the future to log which key successfully decrypted so that we know when we can retire an older key. As it stands, we have to do math repeatedly on older keys.

I was creating a system that used a key index prefixed to the encrypted block to designate which key was associated. That way we can track key usage. I had a few sticking points and got clever than I should have, so I reverted to just the simple approach.